### ISSUE TITLE
"Full Screen" console

### ISSUE THREAD

**Issue by lukencode** [lukencode/NetBash#22]
Maybe use the /netbash route for a nice big full screen console - make it easier to view big results.

**Comment by dkarzon** [comment]
would be pretty cool, then maybe give commands the option to be available in both or only 1? Or at least give them the context?

**Comment by lukencode** [comment]
yeah context could do, dont really want to break existing commands though
